api/base/throttling.py

```diff
@@ -1,15 +1,40 @@
+from rest_framework.settings import api_settings as drf_settings
 from rest_framework.throttling import UserRateThrottle, AnonRateThrottle, SimpleRateThrottle
 
+from api.base import settings
+
 
 class BaseThrottle(SimpleRateThrottle):
 
     def failure(self, request):
         return False
 
+    def get_ident(self, request):
```

Comment:
Not a blocker, but this could be rewritten to check for `HTTP_X_THROTTLE_TOKEN` first and return the `super` call if it's absent, as this has a three line diff from the [base implementation](https://github.com/tomchristie/django-rest-framework/blob/a68b37d8bc432fae37ef5880aec500002b59f565/rest_framework/throttling.py#L26). Something like:

```python
def get_ident(self, request):
    if request.META.get('HTTP_X_THROTTLE_TOKEN'):
        return request.META['HTTP_X_THROTTLE_TOKEN']
    return super(BaseThrottle, self).get_ident(request)
```